src/styles/common.scss

```diff
@@ -1,6 +1,6 @@
 .container {
   margin: auto;
-  max-width: 80%;
+  max-width: 60%;
```

Comment:
why?

Reply:
for a better view according to the Figma design. If I remove this, then it holds the full width of the screen and looks ugly.